[2024-05-17T09:21:43.592+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: temp_humi_dag.save_data_into_db manual__2024-05-17T09:16:31.875066+00:00 [queued]>
[2024-05-17T09:21:43.600+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: temp_humi_dag.save_data_into_db manual__2024-05-17T09:16:31.875066+00:00 [queued]>
[2024-05-17T09:21:43.601+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 4
[2024-05-17T09:21:43.609+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): save_data_into_db> on 2024-05-17 09:16:31.875066+00:00
[2024-05-17T09:21:43.616+0000] {standard_task_runner.py:60} INFO - Started process 17262 to run task
[2024-05-17T09:21:43.619+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'temp_humi_dag', 'save_data_into_db', 'manual__2024-05-17T09:16:31.875066+00:00', '--job-id', '230', '--raw', '--subdir', 'DAGS_FOLDER/ICTTemp_Humi_DAG.py', '--cfg-path', '/tmp/tmp10ml3iot']
[2024-05-17T09:21:43.621+0000] {standard_task_runner.py:88} INFO - Job 230: Subtask save_data_into_db
[2024-05-17T09:21:43.651+0000] {task_command.py:423} INFO - Running <TaskInstance: temp_humi_dag.save_data_into_db manual__2024-05-17T09:16:31.875066+00:00 [running]> on host e3e2dc6bc2be
[2024-05-17T09:21:43.697+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Group4' AIRFLOW_CTX_DAG_ID='temp_humi_dag' AIRFLOW_CTX_TASK_ID='save_data_into_db' AIRFLOW_CTX_EXECUTION_DATE='2024-05-17T09:16:31.875066+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-17T09:16:31.875066+00:00'
[2024-05-17T09:21:43.704+0000] {logging_mixin.py:188} INFO - Error: 'NoneType' object is not iterable
[2024-05-17T09:21:43.705+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-05-17T09:21:43.711+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=temp_humi_dag, task_id=save_data_into_db, execution_date=20240517T091631, start_date=20240517T092143, end_date=20240517T092143
[2024-05-17T09:21:43.753+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-05-17T09:21:43.770+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
